src/main/webapp/resources/js/pages/projects/share/index.js

```diff
@@ -130,61 +148,82 @@ function ShareApp() {
   return (
     <Row>
       <Col xl={{ span: 18, offset: 3 }} xs={24}>
-        <Card>
-          <PageHeader
-            ghost={false}
-            title={i18n("ShareSamples.title")}
-            onBack={goToPrevious}
-          >
-            <Row>
-              <Col span={6}>
-                <Steps
-                  direction="vertical"
-                  current={step}
-                  style={{ height: 400 }}
-                >
-                  {steps.map((step) => (
-                    <Steps.Step key={step.title} title={step.title} />
-                  ))}
-                </Steps>
-              </Col>
-              <Col span={18}>
-                {finished ? (
-                  <ShareSuccess
-                    currentProject={currentProject}
-                    samples={filtered}
-                    removed={remove}
-                    project={targetProject}
-                  />
-                ) : (
-                  <Space direction="vertical" style={{ width: `100%` }}>
-                    {steps[step].component}
-                    {error}
-                    <div
-                      style={{
-                        display: "flex",
-                        justifyContent: "space-between",
-                      }}
+        <Skeleton loading={projectsLoading} active>
+          <Card>
+            <PageHeader
+              ghost={false}
+              title={i18n("ShareSamples.title")}
+              onBack={goToPrevious}
+            >
+              {projects && projects.length === 0 ? (
```

Comment:
I think we can change this to `projects?.length === 0`

Reply:
I like this, nice find :) updated in 483fa813c506a861710628ef8da40288bc501821